### PR TITLE
Hw03 mongodb

### PR DESCRIPTION
**Pull request by Karwan87**
Unable to load database from MongoDB - all connections is working, but unable to load contacts list. I need a help and solutions

### CODE REVIEW

File: models/contacts.js [hunk 1]
```diff
@@ -1,67 +1,96 @@
...
+// };
+// const writeContactsFile = async (data) => {
+//   try {
+//     await fs.writeFile(contactsFilePath, JSON.stringify(data, null, 2));
+//   } catch (error) {
+//     throw new Error("Error writing contacts file");
+//   }
+// };
+
+// const addContact = async (body) => {
```
Comment (AdamChodowiecGoIt91) [inline models/contacts.js]: please remove commented code ;)

File: models/contacts.js [hunk 2]
```diff
@@ -1,67 +1,96 @@
...
+//   return updatedContact;
+// };
+
+// working on mongoDB
+const listContacts = async () => {
   try {
-    const content = await fs.readFile(contactsFilePath, "utf-8");
-    return JSON.parse(content);
+    const contacts = await Contact.find();
+    console.log(contacts);
```
Comment (AdamChodowiecGoIt91) [inline models/contacts.js]: remove unnecessary console.logs

File: routes/api/contacts.js [hunk 3]
```diff
@@ -1,56 +1,132 @@
...
-const router = express.Router();
+// const Joi = require("joi");
+// working with JSON file
 
-const contactSchema = Joi.object({
-  name: Joi.string().required(),
-  email: Joi.string().email().required(),
-  phone: Joi.string().required(),
-});
+// const contactSchema = Joi.object({
```
Comment (AdamChodowiecGoIt91) [inline routes/api/contacts.js]: You still should use `Joi` to validate payload for `post / patch / put` requests

File: server.js [hunk 4]
```diff
@@ -1,5 +1,12 @@
-const app = require('./app')
+// const express = require("express");
+const app = require("./app");
+// const port = process.env.PORT || 3000;
```
Comment (AdamChodowiecGoIt91) [inline server.js]: commented code

File: models/contacts.js (hunk 1 above)
Comment (Karwan87) [reply models/contacts.js]: Done

File: models/contacts.js (hunk 2 above)
Comment (Karwan87) [reply models/contacts.js]: Done

File: routes/api/contacts.js (hunk 3 above)
Comment (Karwan87) [reply routes/api/contacts.js]: I use the contact schema in contactModel.js - with mongoose :)

File: server.js (hunk 4 above)
Comment (Karwan87) [reply server.js]: Done

**Pull request merged**